src/components/ArticleCard.jsx

```diff
@@ -1,4 +1,6 @@
+import { Outlet, Link } from "react-router-dom";
```

Comment:
are you using outlet yet?

Reply:
> 

Thanks for the catch on grey. Completely missed that! Will sort out loading as well :)